recipe/conda_build_config.yaml

```diff
@@ -5,3 +5,6 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
 folly_build_ext:
   - None
   - "jemalloc"  # [not win]
+fmt:
+  - "10"
+  - "9"
```

Comment:
Why do we need them? I deleted them because their usage was unclear. If there would be a comment, we would not accidentally delete them the next time.

Reply:
I should have put a reason here. fmt 10 was released a month ago, and some libraries using folly might require fmt < 10 (because part of the API has changed).

Indicating `fmt==9.0` allows those libraries to use the latest versions of folly.

I do not know what's the policy of conda-forge, but I think we can remove those three lines.

Reply:
Happy to keep them if they help.